corehq/apps/linked_domain/tests/test_decorators.py

```diff
@@ -75,31 +102,69 @@ def test_returns_false_if_no_domain(self):
         self.assertFalse(result)
 
     def test_returns_true_if_domain_has_release_management_privilege(self):
+        def mock_handler(domain, privilege):
```

Comment:
What does this change do? I'm not that familiar with mock, and I'm surprised that a method used as a "side effect" returns a value.

Reply:
Naming could likely be better, but I also don't want to name it too similar to what the patched method is (mock_domain_has_privilege). The `mock_handler` definition represents the mocked implementation for `domain_has_privilege`. Rather than always return true, I only want to return true if the privilege being checked is the lite version. This is simulating a domain that has the lite version enabled, but not the full version.

Here's a helpful snippet from [mock docs](https://docs.python.org/3/library/unittest.mock.html)
> _side_effect_: A function to be called whenever the Mock is called. See the side_effect attribute. Useful for raising exceptions or dynamically changing return values.